### DISCUSSION TITLE
Add a way to prioritize defaulted sound device on startup.

### DISCUSSION THREAD

**Discussion by angelomcordero** [Belphemur/SoundSwitch#1345] [FAQ]
Every time I start, I need to switch my audio device back. Is there a way to do this in which I'm not aware? As of right now, I think it defaults to the highest alphabetical order of selected devices. So for example, my (G)ame DAC is always selected over my (R)ealtek ... Please and thank you so much for you work!

**Comment by jacobrshields** [comment]
You can add a profile with a trigger of "On Startup".

Example:

![profiles](https://github.com/user-attachments/assets/749f4e09-8ea4-4317-bc96-4fbc1f0489bf)

![add-profile](https://github.com/user-attachments/assets/89f3916e-99d4-47ca-a715-9b7498142d3f)

![profiles-after](https://github.com/user-attachments/assets/7a515330-fd46-4415-86f0-ae2df7787615)
